src/Microsoft.Sbom.Api/Executors/IJsonSerializationStrategy.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Sbom.Api.Entities;
+using Microsoft.Sbom.Extensions;
+
+namespace Microsoft.Sbom.Api.Workflows.Helpers;
+
+public interface IJsonSerializationStrategy
```

Comment:
Let's make `JsonSerializationStrategyFactory`, `Spdx3SerializationStrategy`, `Spdx2SerializationStrategy`, and `IJsonSerializationStrategy` all `internal` unless we need them for unit tests. They shouldn't be part of the public API.
```suggestion
internal interface IJsonSerializationStrategy
```